api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -578,4 +578,10 @@ public List<User> getUsers(String name, List<Role> roles, boolean includeRetired
 	 * @since 2.3.6, 2.4.6, 2.5.4, 2.6.0
 	 */
 	Locale getDefaultLocaleForUser(User user);
+
+	/**
+	 * @param user the user to retrieve last-login-time
+	 * @return timestamp representing last login time
+	 */
+	String getLastLoginTime(User user);
```

Comment:
Did you get a chance to look at this? https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#JavaConventions-NewPublicMethodsandClasses